### ISSUE TITLE
Forgiving Void

### ISSUE THREAD

**Issue by Roguem00n** [Darkosto/SkyFactory4Issues#384]
Forgiving Void is just simply not working, I fall into the void and die.
Respawn as a ghost and open my gravestone with the key.
Sometimes I fall and my stuff in stuck down near the bottom of the void.

I don't reset and fall from the sky like my friends do. Mod works for my friends. But not for me. Really don't know why.

**Comment by Roguem00n** [comment]
Yes I have tried going into the config of the mod and setting everything to true.
Everything was already set to true (besides 2 things which were false) which were like blacklistedwhitelist 
So I tried false
Still died
Don't really know why I am just dying instead of falling.

**Comment by Small-Ku** [comment]
I fixed that by giving myself the stage that called "forgiving_void"
